### PR TITLE
[MRG] Evoked.plot_topo.

### PR DESCRIPTION
**Pull request by jaeilepp**
Just chained the function calls for evoked.plot_topo. https://github.com/mne-tools/mne-python/issues/2240

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -284,6 +285,75 @@ def plot_evoked(evoked, picks=None, exclude='bads', unit=True, show=True,
...
+    fig : Instance of matplotlib.figure.Figure
+        Images of evoked responses at sensor locations
+    """
+    return plot_topo(evoked=evoked, layout=layout, layout_scale=layout_scale,
+                     color=color, border=border, ylim=ylim, scalings=scalings,
+                     title=title, proj=proj, vline=vline,
+                     fig_facecolor=fig_facecolor,
+                     fig_background=fig_background,
+                     axis_facecolor=axis_facecolor, font_color=font_color,
+                     show=show)
```
Comment (agramfort) [inline mne/viz/evoked.py]: shall we deprecate plot_topo in favor of plot_evoked_topo then?

Comment (dengemann) [inline mne/viz/evoked.py]: why not ...

File: mne/viz/evoked.py [hunk 2]
```diff
@@ -284,6 +285,75 @@ def plot_evoked(evoked, picks=None, exclude='bads', unit=True, show=True,
...
+                     fig_background=None, axis_facecolor='k', font_color='w',
+                     show=True):
+    """Plot 2D topography of evoked responses.
+
+    Clicking on the plot of an individual sensor opens a new figure showing
+    the evoked response for the selected sensor.
+
+    Parameters
+    ----------
+    evoked : list of Evoked | Evoked
```
Comment (mainakjas) [inline mne/viz/evoked.py]: should we call this `evokeds` ?

Comment (agramfort) [inline mne/viz/evoked.py]: +/- 0 on this

Comment (Eric89GXL) [inline mne/viz/evoked.py]: I think usually we use singular

Comment (dengemann) [inline mne/viz/evoked.py]: +/-0

**Comment by mainakjas** [comment]
@jaeilepp PR looks clean to me. +1 for merge once the deprecation is taken care of

**Comment by jaeilepp** [comment]
plot_evoked_topo is calling the plot_topo function. So the deprecation won't be that straight forward. It would need some refactoring. Should I do that?

**Comment by mainakjas** [comment]
yep, go ahead with the refactoring

**Comment by agramfort** [comment]
+1

it means renaming plot_topo to plot_evoked_topo than adding back a plot_topo that calls plot_evoked_topo while raising a deprecation warning.

**Comment by jaeilepp** [comment]
Ready for review.

**Comment by kingjr** [comment]
Could we add `axis` parameters ?

File: examples/visualization/plot_topo_compare_conditions.py [hunk 3]
```diff
@@ -21,7 +21,7 @@
 import mne
 
 from mne.io import Raw
-from mne.viz import plot_topo
+from mne.viz.evoked import plot_evoked_topo
```
Comment (agramfort) [inline examples/visualization/plot_topo_compare_conditions.py]: don't import from evoked.

from mne.viz import plot_evoked_topo

File: doc/source/whats_new.rst [hunk 4]
```diff
@@ -12,6 +12,8 @@ Changelog
 
     - Add epochs browser to interactively view and manipulate epochs by `Jaakko Leppakangas`_
 
+    - Add ``plot_topo`` method to ``Evoked`` object by `Jaakko Leppakangas`_
+
```
Comment (agramfort) [inline doc/source/whats_new.rst]: document the deprecation

**Comment by agramfort** [comment]
besides LGTM

I fine with axis param if it works but maybe in another PR

File: doc/source/whats_new.rst [hunk 5]
```diff
@@ -12,6 +12,8 @@ Changelog
 
     - Add epochs browser to interactively view and manipulate epochs by `Jaakko Leppakangas`_
 
+    - Add ``plot_topo`` method to ``Evoked`` object which deprecates ``topo.plot_topo`` by `Jaakko Leppakangas`_
```
Comment (mainakjas) [inline doc/source/whats_new.rst]: `topo.plot_topo`, what is that?

**Comment by mainakjas** [comment]
besides LGTM

File: doc/source/whats_new.rst (hunk 5 above)
Comment (jaeilepp) [inline doc/source/whats_new.rst]: The deprecated plot_topo is in topo module. I don't know how I can refer to it otherwise.

Comment (agramfort) [inline doc/source/whats_new.rst]: say mne.viz.plot_topo

**Comment by jaeilepp** [comment]
Ready for merge?

File: doc/source/whats_new.rst [hunk 6]
```diff
@@ -18,6 +18,8 @@ Changelog
 
     - Add support for scaling and adjusting the number of channels/time per view by `Jaakko Leppakangas`_
 
+    - Add ``plot_topo`` method to ``Evoked`` object which deprecates ``mne.viz.plot_topo`` by `Jaakko Leppakangas`_
```
Comment (agramfort) [inline doc/source/whats_new.rst]: this is not true. I would write:

Add ``plot_topo`` method to ``Evoked`` object by `Jaakko Leppakangas`_

and in an API section.

Rename and deprecate ``mne.viz.plot_topo`` for ``mne.viz.plot_evoked_topo`` by `Jaakko Leppakangas`_

**Comment by agramfort** [comment]
besides LGTM

**Comment by jaeilepp** [comment]
Ready. I'll tackle https://github.com/mne-tools/mne-python/issues/2255 during the sprint.

File: doc/source/whats_new.rst [hunk 7]
```diff
@@ -30,6 +32,12 @@ BUG
 
     - Fix picking channels from forward operator could return a channel ordering different from ``info['chs']`` by `Chris Bailey`_
 
+
+API
+~~~
+
+    - Rename and deprecate ``mne.viz.plot_topo`` for ``mne.viz.plot_evoked_topo`` by `Jaakko Leppakangas`_
```
Comment (teonlamont) [inline doc/source/whats_new.rst]: can now be 
```
:func:`mne.viz.plot_topo`
```
etc.

**Comment by mainakjas** [comment]
@jaeilepp why isn't this merged yet? can you rebase?

**Comment by mainakjas** [comment]
@agramfort @dengemann do we have the green signal?

**Comment by agramfort** [comment]
Yes If travis error is unrelated

**Comment by jaeilepp** [comment]
Fixed bad indentation. Should be okay now.

**Comment by mainakjas** [comment]
scrutinizer is not happy, or did it fail for some unrelated reason?

**Comment by jaeilepp** [comment]
If I understand correctly it just failed for some reason. Seemed to be happy before I fixed the indentation.

**Comment by agramfort** [comment]
Thanks

**Comment by mainakjas** [comment]
awesome thanks @jaeilepp !

**Pull request merged**